js/app.js

```diff
@@ -48,4 +48,25 @@ btnSubmit.addEventListener('click', (e) => {
     error.classList.remove('.alert');
     error.classList.add('.off-alert');
   }, 100);
-});
\ No newline at end of file
+});
+
+// storage
+
+const userName = document.getElementById('fullname');
+const userEmail = document.getElementById('email');
+const userMsg = document.getElementById('message');
+
+changeHandler = () => {
+  const userData = {
+    name: userName.value,
+    email: userEmail.value,
+    msg: userMsg.value,
+  };
+  localStorage.setItem('userInfo', JSON.stringify(userData));
+};
+
+const user = JSON.parse(window.localStorage.getItem('userInfo'));
+
+userName.value = user.name;
+userEmail.value = user.email;
+userMsg.value = user.msg;
```

Comment:
- [ ] Please as the preivous code reveiwer asked make the functionality works well since I can't see any data on thr localstorage you can check it from your dev tools 🙏🏻🙏🏻